Library/Formula/netperf.rb

```diff
@@ -11,6 +11,12 @@ class Netperf < Formula
     sha256 "4e0aa192670d1de341e77e0bd3af4b2cccc6a5b12a734e922e92c7b7d50f281d" => :mountain_lion
   end
 
+  # patch fixes demo mode which is required by a lot of software that depends on netperf
+  patch :p2 do
+    url "https://raw.githubusercontent.com/trelane/netperf-brew-patch/master/netlib.diff"
```

Comment:
I'm not saying that we won't accept it, I'm suggesting we bump the formula `revision` so _everyone_ gets the fix now rather than whenever there's next an upstream update or they do a manual `reinstall`, etc.

If you could add a notation that you emailed x upstream developer on x date that'd be helpful too, if there's no public copy of that mail.